service/.env

```diff
@@ -0,0 +1,2 @@
+MAVENLINK_TASK_URL="https://api.mavenlink.com/api/v1/stories"
+MAVENLINK_TOKEN="0fb05785d32d00eb4278c5012f5d3682361862ad680a3e925da42b50a444f681"
```

Comment:
Is this the actual token? 😰 If so, we should remove this from the commit history. 

By a template I meant creating a file called `.env.example` that contains keys from the key-value pair, like `MAVENLINK_TOKEN`, but not with the actual value. So when the other person sets up their environment, they create a `.env` file, copy the content of `.env.example`, paste it in `.env` and write the values of they keys.

Apologies for not being clearer in the other review